src/UserService.Broker/Consumers/UserLoginConsumer.cs

```diff
@@ -27,11 +27,12 @@ public async Task Consume(ConsumeContext<IUserCredentialsRequest> context)
 
         private object GetUserCredentials(IUserCredentialsRequest request)
         {
-            DbUser userCredentials = repository.GetUserByEmail(request.Email);
+            DbUser user = repository.GetUserByEmail(request.Email);
```

Comment:
email now in userCreds, rework this => remove dbUser

Reply:
Fixed